api-specificatie/openapi.yaml

```diff
@@ -1343,22 +1379,35 @@ components:
         ligtAanOpenbareRuimte:
           $ref: '#/components/schemas/NEN3610ID'
           description: 'Een nummeraanduiding ligt aan een openbare ruimte.'
-        huisnummerMogelijkOnjuist:
-          type: boolean
-        huisletterMogelijkOnjuist:
-          type: boolean
-        huisnummertoevoegingMogelijkOnjuist:
-          type: boolean
-        postcodeMogelijkOnjuist:
-          type: boolean
-        typeAdresseerbaarObjectMogelijkOnjuist:
-          type: boolean
-        statusMogelijkOnjuist:
-          type: boolean
-        ligtInWoonplaatsMogelijkOnjuist:
-          type: boolean
-        ligtAanOpenbareRuimteMogelijkOnjuist:
-          type: boolean
+        mogelijkOnjuist:
+          $ref: '#/components/schemas/NummeraanduidingMogelijkOnjuist'
+    NummeraanduidingMogelijkOnjuist:
+        type: object
+        properties: 
+          huisnummerMogelijkOnjuist:
+            type: boolean
+            description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
+          huisletterMogelijkOnjuist:
+            type: boolean
+            description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
+          huisnummertoevoegingMogelijkOnjuist:
+            type: boolean
+            description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
+          postcodeMogelijkOnjuist:
+            type: boolean
+            description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
+          typeAdresseerbaarObjectMogelijkOnjuist:
+            type: boolean
+            description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
+          statusMogelijkOnjuist:
+            type: boolean
+            description: "Indicator die aangeeft of er onderzoek wordt gedaan naar de juistheid van de waarde in het corresponderende gegeven."
+          mogelijkVerkeerdeWoonplaatsGebruikt:
```

Comment:
woonplaats (en hieronder ook straat) heeft maar 1 mogelijke oorzaak voor onjuistheid. Is het dan nog nodig de toevoeging te doen? Ik denk dat dan kan worden volstaan met wat in de description staat

Reply:
Deze heb ik weer aangepast naar ligtInWoonplaats en ligtAanOpenbareRuimte. Het zijn immers die properties uit de Nummeraanduiding die mogelijk onjuist zijn.